index.md

```diff
@@ -719,7 +379,7 @@ According to the Common Specification, any ID element must start with a prefix
 (also, the XML ID data type does not permit IDs that start with a number, so a
 prefix solves this issue).
 
-We recommended to use as a prefix an internationally recognized standard identifier
+We recommend to use as a prefix an internationally recognized standard identifier
 for the institution from which the SIP originates. This may lead to problems
```

Comment:
We recommend to use as a prefix an internationally recognized standard identifier for the institution from which the SIP originates. ==
We recommend using an internationally recognized standard identifier for the institution from which the SIP originates as a prefix.

Reply:
done